src/test/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeTest.java

```diff
@@ -465,19 +465,22 @@ public void trackVideoPlaybackStarted() {
 
     verify(heartbeat).trackSessionStart(isEqualToComparingFieldByFieldRecursively(mediaInfo),
         eq(videoMetadata));
+    assertTrue(integration.playbackDelegate != null);
   }
 
   @Test
   public void trackVideoPlaybackPaused() {
     newVideoSession();
     heartbeatTestFixture("Video Playback Paused");
+    assertTrue(integration.playbackDelegate.isPaused);
     verify(heartbeat).trackPause();
   }
 
   @Test
   public void trackVideoPlaybackResumed() {
     newVideoSession();
     heartbeatTestFixture("Video Playback Resumed");
+    assertTrue(!integration.playbackDelegate.isPaused);
```

Comment:
I think these tests are brittle and easy to regress. It's testing internal implementation rather than behaviour. We could have a bug in the delegate that sets `isPaused` to true|false properly (implementation), but doesn't return the correct playhead positions (behaviour).